ts/error.ts

```diff
@@ -0,0 +1,53 @@
+import { AttributeDefinition, AttributeList } from "./component/attribute-list";
+
+interface ErrorData {
+  status: string;
+  reason: string;
+  message: string;
+}
+
+const errorAttributes: AttributeDefinition<ErrorData>[] = [
+  {
+    title: "Code",
+    addContents: (errorData, fragment) => {
+      fragment.textContent = errorData.status || "N/A";
+    },
+  },
+  {
+    title: "Error",
+    addContents: (errorData, fragment) => {
+      fragment.textContent = errorData.reason;
+    },
+  },
+  {
+    title: "Message",
+    addContents: (errorData, fragment) => {
+      fragment.textContent = errorData.message;
+    },
+  },
+];
+
+document.addEventListener("DOMContentLoaded", () => {
+  const container = document.getElementById("error-attribute-list");
+
+  if (container) {
+    const errorData: ErrorData = {
+      status: container.dataset.status ?? "N/A",
```

Comment:
"Unknown" would make more sense than "N/A"
Can use `||` instead of `??` to take care of empty string too. Same for `reason`